dvc/tree/local.py

```diff
@@ -292,14 +281,15 @@ def _unprotect_dir(self, path):
             self._unprotect_file(fname)
 
     def unprotect(self, path_info):
-        path = path_info.fspath
-        if not os.path.exists(path):
-            raise DvcException(f"can't unprotect non-existing data '{path}'")
```

Comment:
We have a test for broken symlink removal (`dvc.remove()`)that checks for this exception. It should really be ignored on a higher level, not raised in a special exception.